lib/shopify_api/graphql/task.rake

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+require 'fileutils'
+
+namespace :shopify_api do
+  namespace :graphql do
+    prereqs = []
+    # add the Rails environment task as a prerequisite if loaded from a Rails app
+    prereqs << :environment if Rake::Task.task_defined?('environment')
+
+    desc 'Writes the Shopify Admin API GraphQL schema to a local file'
+    task dump: prereqs do
+      site_url = ENV['SITE_URL'] || ENV['site_url']
+      shop_domain = ENV['SHOP_DOMAIN'] || ENV['shop_domain']
+      api_version = ENV['API_VERSION'] || ENV['api_version']
+      access_token = ENV['ACCESS_TOKEN'] || ENV['access_token']
```

Comment:
All of these environment variables are not specified in your error or where to get the values. Maybe this should be part of the README update as well.